tests/indicators/m-r/Mama/Mama.BufferList.Tests.cs

```diff
@@ -103,4 +103,41 @@ public void ClearResetsState()
         sut.Should().HaveCount(expected.Count);
         sut.Should().BeEquivalentTo(expected);
     }
+
+    [TestMethod]
+    public void AutoPruning()
```

Comment:
This test should also ensure result values are correct after pruning, including next incremental values.